src/GameSync.Models/Users/ValueObjects/Password.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace GameSync.Models.Users.ValueObjects;
+public class Password
+{
+    public string Value { get; private set; }
+
+    private Password(string value)
+    {
+        if (string.IsNullOrEmpty(value))
+            throw new ArgumentNullException(nameof(Password));
+
+        if (value.Length < 8)
```

Comment:
Not sure if validations should occur here.

Reply:
I think it depends. Maybe the password length check isn't the best example here, because it isn't the logic (domain) concern, but more an infrastructural concern. It was only for demo that we can check if values are withing a range (or not).